### PR TITLE
Changes disclosure policy for user, viz, permission presenters

### PR DESCRIPTION
**Pull request by gfiorav**
CR please, @juanignaciosl 

We've done some front end tests in staging and everything is fine.

cc/ @xavijam

### CODE REVIEW

File: app/models/carto/permission.rb [hunk 1]
```diff
@@ -41,6 +41,10 @@ def is_permitted?(user, permission_type)
     SORTED_ACCESSES.index(permission_for_user(user)) <= SORTED_ACCESSES.index(permission_type)
   end
 
+  def is_owner_user?(user)
```
Comment (juanignaciosl) [inline app/models/carto/permission.rb]: :+1: to Hound

Comment (juanignaciosl) [inline app/models/carto/permission.rb]: Well, since method already existed, :-1: :sweat_smile:

**Comment by juanignaciosl** [comment]
This lacks a test for the new behaviour.

**Comment by gfiorav** [comment]
@juanignaciosl you're right... Adding. I'll ping you when done, thanks!

**Comment by gfiorav** [comment]
Retest this please

**Pull request merged**